### DISCUSSION TITLE
zksync plugin problem

### DISCUSSION THREAD

**Discussion by imranshafiq55** [Cyfrin/foundry-full-course-cu#2993] [Q&A]
Hi,
Zksync plugin is not fetching my file to compile.
[
![zksync deploy](https://github.com/user-attachments/assets/adae7ea1-bdd3-49c7-bca0-285126483c9d)
](url)

**Comment by heheboii11** [comment]
Just dont stress too much with zksync if things are not working, just use normal sepolia

**Comment by EngrPips** [comment]
Hello @imranshafiq55, I created awareness with Patrick to confirm that this is not an issue from the `ZKSync` team end. We will get back as soon as possible pending the response I will say continue with the course and don't let this block you.

**Comment by PatrickAlphaC** [comment]
Hi @imranshafiq55, can you share what your files and folders look like? And what exact button you got to get this?

**Comment by Divyanshu-20** [comment]
I think you need to create Folder named "contracts" & add your .sol file in that. It worked for me.

**Reply by EngrPips** [reply]
Thanks for the heads-up.
